.github/workflows/benchmark/terraform/variables.tf

```diff
@@ -10,13 +10,13 @@ variable "secret_key" {
 
 variable "region" {
   type    = string
-  default = "ap-northeast-1"
+  default = "us-east-2"
   description = "AWS region"
 }
 
 variable "instances_count" {
   type    = number
-  default = 3
+  default = 1
```

Comment:
CKB has some issues with network synchronization at the moment. I'm just benching one node here.